doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -676,25 +678,25 @@ PyArrayDescr_Type
 
 
 The :c:data:`PyArray_Type` typeobject implements many of the features of
-Python objects including the tp_as_number, tp_as_sequence,
-tp_as_mapping, and tp_as_buffer interfaces. The rich comparison
-(tp_richcompare) is also used along with new-style attribute lookup
-for methods (tp_methods) and properties (tp_getset). The
+Python objects including the `tp_as_number`, `tp_as_sequence`,
+`tp_as_mapping`, and `tp_as_buffer` interfaces. The rich comparison
+(`tp_richcompare`) is also used along with new-style attribute lookup
+for methods (`tp_methods`) and properties (`tp_getset`). The
 :c:data:`PyArray_Type` can also be sub-typed.
```

Comment:
The changes are not being rendered as links, I am not sure what sphinx construct you need to use. Perhaps 
```
:c:data`tp_as_number`
```
Looking at the [link you wish to create](https://docs.python.org/3/c-api/typeobj.html?highlight=tp_as_number#c.tp_as_number) I see it has a `c.tp_as_number` identifier, so maybe
```
:py:`c.tp_as_number`
```
would work

Reply:
@mattip the circleci is failing due to unrecognised `:py:`. Where should it be added so that this doesn't throw an error. Also, does it recognise `:c:` by default, no right?

Reply:
I dug around in the [intersphinx index for python](https://docs.python.org/dev/objects.inv) (I downloaded the file, then in vim removed the top 4-5 lines until the binary data started, then did `cat ~/Downloads/objects.inv | zlib-flate -uncompress |grep tp_member` to find the link info for `tp_member`). Since this is rather advanced, I fixed the links and pushed a commit.